source/_docs/blueprint/schema.markdown

```diff
@@ -3,27 +3,88 @@ title: "Blueprint schema"
 description: "The schema for a valid blueprint."
 ---
 
-Schema of the blueprint metadata:
-
-```ts
-interface BlueprintInput {
-  name?: string;
-  description?: string;
-  selector?: Selector;
-  default?: any;
-}
-
-interface Blueprint {
-  blueprint: {
-    domain: string;
-    name: string;
-    input?: Record<string, BlueprintInput | null>;
-    description?: string;
-    source_url?: string;
-  }
-}
+### Schema of the blueprint metadata
+
+<div class='note'>
+
+Values inside `<>` are required, inside `[]` are optional.
```

Comment:
We should not all of a sudden introduce new standards.

We have developer documentation on how to document YAML and defined standards for placeholders. We also have helpers for showing YAML properties, their types, if they are optional or not and provide a description for each of those.

If anything, we should stick to a standard. I'm happy to improve standards, but that should be done site wide.

Reply:
I can fully agree with you, thank you for looking into this.